src/main/java/hu/oe/nik/szfmv/Main.java

```diff
@@ -35,14 +56,62 @@ public static void main(String[] args) {
         gui.getCourseDisplay().drawWorld(w);
 
         while (true) {
-            try {
-                car.drive();
-                gui.getCourseDisplay().drawWorld(w);
-                Thread.sleep(CYCLE_PERIOD);
-            } catch (InterruptedException e) {
-                LOGGER.error(e.getMessage());
+            car.drive();
+
+            gui.getCourseDisplay().drawWorld(w);
+        }
+    }
+    private static ArrayList<WorldObject> roads = new ArrayList<WorldObject>();
+
+    private static void loadRoads(){
+        String filePath = "src\\main\\resources\\test_world.xml";
+        File xmlFile = new File(filePath);
```

Comment:
```suggestion
        File xmlFile = new File(ClassLoader.getSystemResource("test_world.xml").getFile());
```
Használjátok a ClassLoadert, mert simán lehet, hogy az ilyenek miatt nem tudtok demózni. Az én gépemen más OS van...

Reply:
Ez a két sor azért maradt csak benne, mert nekünk volt teszt, hogy lássuk, kirajzolódik-e a világ (a main conflict-jának feloldása során ki fog kerülni, amit rövidesen meg is csinálunk), de a referencia point beolvasása már a classLoader alapján történik.